docs/wallets/cli/dcrwallet-tickets.md

```diff
@@ -70,79 +65,46 @@ vsp.pubkey=ia9Ra2Drb+OHLqRyBsJnRKBd7TUG1IvrseC6robKzGo=
 
 ---
 
-## Ticket Purchasing
+## Step 2. Purchase Tickets
+
+Tickets can be purchased in two different ways, manual or automatic. Purchasing
```

Comment:
Isn't it true that manual ticket buying will never be mixed, but automatic ticket buying CAN be mixed if you've configured with cspp? Or is automatic ticket buying ALWAYS mixed via cspp?

Either way, I think those details should be briefly mentioned here.

Reply:
`how-to-cspp.md` explains how to set up mixed ticket buying. I will add a link.

Reply:
I also don't know if manual (`dcrctl --wallet purchaseticket`) can mix but the autobuyer can be used *without* mixing.